rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @book.rentals.push(self)
+    @person = person
+    @book.rentals.push(self)
```

Comment:
- [ ] Kindly add a belongs-to association between a person and a rental to indicate that a rental belongs to a person, just as a rental belongs to a book.